app/proto/requests.proto

```diff
@@ -79,6 +83,10 @@ message RespondHostRequestReq {
   string text = 3;
 }
 
+message ArchiveHostRequestReq {
+  int64 host_request_id = 1;
```

Comment:
We should also have an API for unarchiving. Maybe easiest to change this to `SetHostRequestArchiveStatus` and then add a `bool set_archived = 2;` here.